subprojects/core/src/test/groovy/org/gradle/initialization/buildsrc/BuildSrcUpdateFactoryTest.groovy

```diff
@@ -30,7 +30,7 @@ class BuildSrcUpdateFactoryTest extends Specification {
     def launcher = Stub(BuildController)
     def listener = Stub(BuildSrcBuildListenerFactory.Listener)
     def listenerFactory = Mock(BuildSrcBuildListenerFactory)
-    def factory = new BuildSrcUpdateFactory(cache, launcher, listenerFactory)
```

Comment:
The `cache` field and all the configuration belonging to it can be removed.

Reply:
removed.